src/pgduckdb_planner.cpp

```diff
@@ -63,6 +64,9 @@ CreatePlan(Query *query, const char *query_string, ParamListInfo bound_params) {
 
 	PlannerInfo *query_planner_info = PlanQuery(query, bound_params);
 	auto duckdb_connection = pgduckdb::DuckdbCreateConnection(rtables, query_planner_info, vars, query_string);
+	if (!duckdb_connection) {
```

Comment:
In what cases is this expected to happen? Afaict when reading the `DuckdbCreateConnection` function this should not be possible.

Reply:
I made a modification to `DuckdbCreateConnection` so it could return nullptr, but it didn't stick - I'll remove this branch 👍